xmtp_mls/src/groups/mod.rs

```diff
@@ -2911,8 +2907,7 @@ pub(crate) mod tests {
         );
     }
 
-    #[cfg(not(target_arch = "wasm32"))]
```

Comment:
be careful replacing this with xmtp_common test, since this cfg is to not run the test at all in wasm32, rather than switch the test executor to `wasm_bindgen::test`. Since this test uses environment variables it cannot work in webassembly